untitled/src/test/java/ManTest.java

```diff
@@ -0,0 +1,42 @@
+import org.example.Man;
+import org.example.Woman;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+public class ManTest {
+
+        private Man man;
+        private Woman woman;
+
+        @BeforeMethod
+        public void setUp() {
+            man = new Man("John", "Doe", 70, null);
+            woman = new Woman("Jane", "Smith", 65, null, "Smith");
```

Comment:
For testing that men is retired you need to use boundary values and negative testign as well, for men age values: 64, 65, 66. For woman: 59,  60, 61